tutorials/profiler/generate.py

```diff
@@ -0,0 +1,339 @@
+import pytest
```

Comment:
nit: probably better to rename the file itself to something that includes `profiler` in the name as well, otherwise it will be a bit confusing I think